client/src/actions.js

```diff
@@ -0,0 +1,32 @@
+let socket
+let _initialized = false
+
+function initialized() {
+    return _initialized
```

Comment:
This seems a little gross, but I don't understand enough about what's going on here to suggest an alternative